analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output.yml

```diff
@@ -149,5 +149,5 @@ issues:
 - timestamp: "1970-01-01T00:00:00Z"
   source: "NuGet"
   message: "Failed to get package data for 'NuGet::foobar:1.2.3': IOException: Failed\
-    \ to get a response body from 'https://api.nuget.org/v3/registration5-gz-semver2/foobar/1.2.3.json'."
+    \ to retrieve package data for 'foobar' from any of [https://api.nuget.org/v3/registration5-gz-semver2]."
```

Comment:
Sorry, one more thing: This file should not be modified as part of this commit, or?

Reply:
It has to be modified as part of the change in `NuGetSupport.kt`, which is already used by the DotNet package manager.
This commit changed the changed the error message and would fail the `DotNetFunTest` if it would not be fixed here.